src/asynk.rs

```diff
@@ -636,6 +636,14 @@ impl Options {
         }
     }
 
+    /// Select option to include retry logic with backoff 
```

Comment:
Would be good to update the docs and examples so they are on the same level as other options.

Go doc for this option is nice example: https://github.com/nats-io/nats.go/blob/main/nats.go#L449
(Rust client documents Options methods rather than properties)

Reply:
Updated as requested.